src/MPS.jl

```diff
@@ -504,18 +504,30 @@ end
 # TODO dispatch on form
 # TODO generalize to AbstractAnsatz
 function mixed_canonize!(tn::AbstractMPO, orthog_center)
+    if orthog_center isa Site
+        left = id(orthog_center) - 1
+        right = id(orthog_center) + 1
+    else
+        values = [id(site) for site in orthog_center]
+        orthog_center = Vector{Site}(orthog_center)
+
+        left, right = extrema(values) .+ (-1, 1)
+    end
+
     # left-to-right QR sweep (left-canonical tensors)
-    for i in 1:(id(orthog_center) - 1)
+    for i in 1:left
         canonize_site!(tn, Site(i); direction=:right, method=:qr)
     end
 
     # right-to-left QR sweep (right-canonical tensors)
-    for i in nsites(tn):-1:(id(orthog_center) + 1)
+    for i in nsites(tn):-1:right
         canonize_site!(tn, Site(i); direction=:left, method=:qr)
     end
 
     # center SVD sweep to get singular values
-    # canonize_site!(tn, orthog_center; direction=:left, method=:svd)
+    # for i in (left + 1):(right - 1)
+    #     canonize_site!(tn, Site(i); direction=:left, method=:svd)
+    # end
```

Comment:
with the orthogonality center not being a unique site, this makes little sense right?

what do you think? should we remove this comment and later move it to the `BondMixedCanonical`?

Reply:
yes, for me it is okay to remove the comment! I will do that tomorrow in a commit :)